.vscode/launch.json

```diff
@@ -14,8 +14,8 @@
             "program": "${workspaceFolder}/api/index.js",
             "envFile": "${workspaceFolder}/api/.env",
             "env": {
-                "WHITELIST_ORIGIN": "http://localhost:4000",
-                // "RUN_ONLY_NETWORK": "cosmos-hub-testnet"
```

Comment:
should this be in the .vscode file?

Reply:
OK, I just deleted this whole `env` block from the `launch.json`